plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtVMDef.java

```diff
@@ -1191,6 +1196,9 @@ public String toString() {
             if (_networkName != null) {
                 netBuilder.append("<target dev='" + _networkName + "'/>\n");
             }
+            if (_mtu > 0 && _mtu <= 9000) {
```

Comment:
I see you silently skip the MTU if it's equal to 9000, but some networks also allow 9216 as that's what the switch allows.

I'm also not seeing why we should allow an MTU of 10 or something. Shouldn't 1280 be the minimum?

And shouldn't the API do this check already?